crates/sui/src/client_commands.rs

```diff
@@ -905,6 +930,32 @@ impl SuiClientCommands {
                 context.config.envs.clone(),
                 context.config.active_env.clone(),
             ),
+            SuiClientCommands::VerifySource {
+                package_path,
+                build_config,
+                no_deps,
+                addr,
+            } => {
+                let compiled_package = build_move_package(
+                    &package_path,
+                    BuildConfig {
+                        config: build_config,
+                        run_bytecode_verifier: true,
+                        print_diags_to_stderr: true,
+                    },
+                )?;
+
+                let client = context.get_client().await?;
+                SuiClientCommandResult::VerifySource(
+                    match BytecodeSourceVerifier::new(client.read_api(), false)
+                        .verify_package(&compiled_package.package, !no_deps, addr.map(|q| q.into()))
+                        .await
+                    {
+                        Ok(_) => "Ok".to_string(),
+                        Err(e) => format!("{}", e),
```

Comment:
Do we have an ability to set a non-zero exit code when this happens?

Reply:
Also, let's colourise the output, to make it extra clear.

Reply:
> Do we have an ability to set a non-zero exit code when this happens?

I tried [this](https://crates.io/crates/exitcode) but it also terminate the process. So no clean way to do this without reworking how we handle the command logic. Ideally the error should be propagated all the way up but thats a different work item.